scripts/gulp/frontend-shared.js

```diff
@@ -15,7 +15,7 @@ const buildFrontendSharedJs = () => {
 
   return (
     gulp
-      .src('frontend-shared/src/*')
+      .src('frontend-shared/src/**/*.js')
```

Comment:
This makes sure that files in subdirectories of `frontend-shared/src` are properly processed.

Reply:
But will that transpile tests too?

Reply:
Unrelated, I think we may have an issue with source maps not working in the debugger, I need to follow up on that. It might be a path issue.. or I'm mistaken and they do work.

Reply:
> But will that transpile tests too?

Ah, yes it will. I will fix that.